### ISSUE TITLE
PoissonReconstrcution.exe doesn't work

### ISSUE THREAD

**Issue by charlieputhedu** [mkazhdan/PoissonRecon#112]
I download the exe from ReadMe-Links-Executables-Win64 package, however it doesn't work when I am using the PoissonRencon.exe, just like the figure shows. I don't know why, the same parameter works on the old version and looks good however it doesn't works on the lastest version. Figure2 shows the parameters what I used and I use VS2015+QT5.9.1+vtk8.1.1.
![1560320717(1)](https://user-images.githubusercontent.com/51358474/59328990-5a011100-8d20-11e9-90aa-cd2cad03e9a7.jpg)
![image](https://user-images.githubusercontent.com/51358474/59329151-c845d380-8d20-11e9-9c80-0d2a85f9f6ae.png)

![1560322212(1)](https://user-images.githubusercontent.com/51358474/59329426-69cd2500-8d21-11e9-95a4-7ef5e6085d0e.jpg)

**Comment by mkazhdan** [comment]
Unfortunately, I am not able to parse the error you are receiving.

**Comment by charlieputhedu** [comment]
The input file format is a txt which contains point cloud like this:
![image](https://user-images.githubusercontent.com/51358474/59352478-5aff6600-8d53-11e9-9673-9b38e6db17bd.png)

**Comment by mkazhdan** [comment]
Definitely looks reasonable. Not sure why the crash is happening. Can you share the pointsets?

**Comment by charlieputhedu** [comment]
Sure, I have generated a zip file containing two folders that are ply generated by the old version and the new version of the Poisson reconstruction program respectively. The point cloud data is the same. We can see that the old version successfully generated the model, while the new version had no model.

[PointSetsAndResults.zip](https://github.com/mkazhdan/PoissonRecon/files/3283785/PointSetsAndResults.zip)

**Comment by mkazhdan** [comment]
The pointset definitely seems fine.
When I run Version 11.02 on my computer, the code runs fine.
Have you tried downloading the latest Visual Studio redistributables?

**Comment by charlieputhedu** [comment]
Nope,the Visual Studio which I am using is the old one. I will have a try. Thanks.Besides, I want to know that if the exe limits the number of points? Because the exe failed when I used the version 5.0 with  31836240 input points(shown in the picture).The things what I have done is model reconstruction of large amount of data. So I'm curious about its feasibility, Thanks again.
![0e6948147ae865dea259357ba1c9e7a](https://user-images.githubusercontent.com/51358474/61295167-05165600-a80a-11e9-8e73-a9a3c0f24c20.jpg)

**Comment by mkazhdan** [comment]
I've used the recent code/executables to reconstruct points sets with billions of points, so you should be ok. 

As for the crashing, you could also try disabling the "Advanced Vector Extensions 2" under project properties (C/C++ -> Code Generation).
